doc/users/install.rst

```diff
@@ -119,7 +125,7 @@ Must Have
 Nibabel_ 1.0 - 1.4
```

Comment:
Should we even list those dependencies? It gives a false impression that user needs to install those libraries manually when in truth all of this is done automatically by pip. I would simplify this to `pip install nipype`